scripts/exercise.js

```diff
@@ -50,8 +57,85 @@ chokidar.watch(exerciseFile).on("all", (event, path) => {
     execSync(`tsc ${exerciseFile} --noEmit --strict`, {
       stdio: "inherit",
     });
-    console.log("Typecheck complete. You finished the exercise!");
+    console.log(`Typecheck complete. You finished the ${exercise} exercise!`);
+  
+    if (isEndOfTutorial(exercise, isSolution, pathIndicator)) {
+      console.log(`\n🎉 Cograts! You've reached to the end of this tutorial! 🎉  \n`);
```

Comment:
```suggestion
      console.log(`\n🎉 Congrats! You've reached the end of this tutorial! 🎉  \n`);
```

Reply:
Nice catch @jhonnymoreira

Reply:
Nice work, @preshetin! 🤜🏻 🤛🏻

Reply:
Cool @jhonnymoreira , I wonder if you had a chance to try it on your local machine. I would really appreciate some validation